src/components/AllArticles.jsx

```diff
@@ -0,0 +1,53 @@
+import Dropdown from "react-bootstrap/Dropdown";
+import { ArticleCard } from "./ArticleCard.jsx";
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+export const AllArticles = () => {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setisLoading] = useState(true);
+
+  useEffect(() => {
+    axios
```

Comment:
I would strongly recommend extracting this to a separate `api.js` file and then making the rest of your api functions in this file moving forwards